### DISCUSSION TITLE
website issues

### DISCUSSION THREAD

**Discussion by KovacsAlex144** [community/community#31947] [Other Feature Feedback, Questions, & Ideas]
why is the link to my website throw back to the menu, why is this happening

https://user-images.githubusercontent.com/112808731/188324453-d5141c80-4084-48c9-b1d6-0d08d55336e2.mp4

**Comment by airtower-luna** [comment]
A video of your screen is not a good way to describe the problem, all I can say from that is that the URL (`github.hu`) looks highly suspicious. It'd be easier to help if you added a link to your repository and described where you expect to see your site.

**Reply by KovacsAlex144** [reply]
Thank you
